spyder/config/main.py

```diff
@@ -357,6 +357,8 @@
               'editor/copy line': "Ctrl+Alt+Down" if WIN else \
                                   "Shift+Alt+Down",
               'editor/delete line': 'Ctrl+D',
+              'editor/upper selected text': 'Ctrl+Shift+U',
+              'editor/lower selected text': 'Ctrl+U',

```

Comment:
`lower selected text` does not sound right

lets change this to 

`transform to uppercase`
`transform to lowercase`